File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
       
        unordered_set<int> mp;

        for(auto num : nums ){
            if(mp.count(num) > 0){
                return true;
            }
            mp.insert(num);
        }

        return false;
        
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        mp = set()
        for i in range(len(nums)):
            if nums[i] in mp:
                return True
            mp.add(nums[i])
        return False